Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -442,6 +448,16 @@ private static void Clean(NugetPackageIdentifier package)
                         break;
                     }
                 }
+                
+                // Checks if Api Compatibility Level in PlayerSettings matches the version requried for this NuGet package.                
+                if (libDirectories.Count() > 0 && selectedDirectories.Count == 0) {
+                    foreach (var directory in libDirectories)
+                    {
+                        string directoryName = directory.Name.ToLower();
+                        // Current API version is NET_Standard_2_0; NET_Standard_4_5 is required to support this NuGet package.
+                        LogVerbose("Current API version is {0}; {1} is required to support this NuGet package.", DotNetVersion, directoryName);
+                    }                                            
+                }
```

Comment:
While I think this might be an error case that should prevent the package from installing, theres no way to shortcut that from this function right now so, I'm hoping you won't mind logging a warning right here about the general issue? Something like "{PackageName} has lib folders, but none supported by the current API Version {DotNetVersion}"?

Please keep the verbose prints, but also log a warning outside of your foreach about the failure to find a compatible lib.